.gitignore

```diff
@@ -6,11 +6,13 @@
 *.fls
 *.log
 *.out
+*.pdf
+*.kate-swp
+*.toc
```

Comment:
`.kate-swp` looks like temporary editor files to me, which I wouldn't like to have in this repo.
Also, delay adding `.pdf` once we remove the pdf.